includes/alerts.inc.php

```diff
@@ -317,6 +317,11 @@ function GetContacts($results)
         }
     }
 
+    # Copy all email alerts to default contact if configured.
+    if ($config['alert']['default_copy'] === true) {
+        $tmp_contacts[$config['alert']['default_mail']] = '';
```

Comment:
Before assigning it to $tmp_contacts, you need to check it doesn't exist already :)